demo/scripts/utils/cssMonitor.ts

```diff
@@ -0,0 +1,54 @@
+import { Stylesheet } from '@fluentui/merge-styles/lib/Stylesheet';
+
+let isCssMonitorStarted: boolean = false;
```

Comment:
what is a css monitor?

Reply:
Fluent UI will dynamically add CSS into the page. When this happens in popout window, we need to copy the new CSS into popout page to make the style correct.